docs/Versioning.md

```diff
@@ -94,6 +94,8 @@ operator id MUST be greater than any extant `op_version` for the
 > 4. Register the new operator in the corresponding `operator_sets`
 >    header file.
 
+### Operator Sets
```

Comment:
I'm not sure it's good to add this line so that a separate "operator sets" section is there. It may mislead readers on getting the points that "Versioning is against an operator set, instead of an operator".

Reply:
That's fair. I do think a clear definition of an `Operator set` is necessary though. Especially given that users who are converting to ONNX from another format often need to specify the opset, and will need to know precisely what this means

Reply:
Agree that a separate section of "OperatorSet and OperatorSet Versioning" should be there. do you want to do another review and refactor of this section "Operator Versioning" and this new "OperatorSet" section please? 

basically, things should be highlighted, 1) Versioning is against OperatorSet, instead of Operator. 2) Operator is identified by tuple <name, domain, since_version> (currently, it's saying "op_version", which is not accurate and misleading). 3) Every ONNX release will at most have 1 OperatorSet version bump (if and only if there's any breaking changes) 4) list the details of "breaking changes".

Reply:
Yep sure, will do!

Reply:
@linkerzhang I created a new Operator set sub-section